packages/boa/requirements.txt

```diff
@@ -1,3 +1,3 @@
 numpy==1.17.4
 jieba==0.42.1
-sklearn==0.0
\ No newline at end of file
+sklearn==0.0
```

Comment:
it is recommended to use scikit-learn and specify specific version instead :)

Reply:
Okay, changed to `scikit-learn@0.23.1`.